project/build.properties

```diff
@@ -0,0 +1 @@
+sbt.version=0.13.9
```

Comment:
intellij was being an ass and giving me a bunch of red squigglies until i did this.